File: GardifyAndroid/app/src/main/java/com/gardify/android/data/settings/UserInfo.kt
package com.gardify.android.data.settings

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UserInfo {
    @Expose
    @SerializedName("Country")
    var country: String? = null

    @Expose
    @SerializedName("Zip")
    var zip: String? = null

    @Expose
    @SerializedName("Street")
    var street: String? = null

    @Expose
    @SerializedName("City")
    var city: String? = null

    @Expose
    @SerializedName("UserName")
    var userName: String? = null

    @Expose
    @SerializedName("LastName")
    var lastName: String? = null

    @Expose
    @SerializedName("FirstName")
    var firstName: String? = null

    @Expose
    @SerializedName("HouseNr")
    var houseNr: String? = null

    @Expose
    @SerializedName("\$id")
    private var `$id`: String? = null

    fun `get$id`(): String? {
        return `$id`
    }

    fun `set$id`(`$id`: String?) {
        this.`$id` = `$id`
    }
}

File: GardifyIOS/GardifyIOS/Models/AppModels/UserInfo.swift
//
//  UserInfo.swift
//  GardifyIOS
//
//  Created by Rifat Hussain on 10.08.20.
//  Copyright © 2020 Netzlab. All rights reserved.
//

import Foundation

struct UserInfo: Codable {
    let HouseNr: String
    let FirstName, LastName, UserName, City: String
    let Street, Zip, Country: String

}

struct UserImageInfo: Codable{
    let Images: [ImageFile]
}

struct UserSettingInfo: Codable{
    let UserId: String
    var ActiveStormAlert, ActiveFrostAlert, AlertByEmail, AlertByPush: Bool
    var FrostDegreeBuffer: Int
}

struct UserWarningModel: Codable{
    
    let Id: Int
    let NotificationType: Int
    let Title, Body: String
    let ConditionValue: Float
    let Date: String
    
}

struct WarningModel : Codable{
    let RelatedObjectId, ObjectType: Int
    var NotifyForWind, NotifyForFrost, IsInPot : Bool
    let Title, Text, RelatedObjectName : String?
    let AlertConditionValue : Decimal
    let Dismissed: Bool
}
